content/en/agent/autodiscovery/troubleshooting.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Autodiscovery Troubleshooting
+kind: documentation
+disable_toc: true
+further_reading:
+- link: "agent/troubleshooting"
+  tag: "Documentation"
+  text: "Agent Troubleshooting"
+- link: "agent/troubleshooting/debug_mode"
+  tag: "Documentation"
+  text: "Agent Debug Mode"
+- link: "agent/troubleshooting/send_a_flare"
+  tag: "Documentation"
+  text: "Send an Agent Flare"
+---
+
+To start troubleshooting the Agent Autodiscovery, run the `configcheck` init script command:
+
+```shell
+docker exec -it <AGENT_CONTAINER_NAME> datadog-agent configcheck -v
+```
+
+**Note**: The `-v` option allows you to see all templates that are loaded but couldn't be resolved:
```

Comment:
To be more accurate it will display **all** templates including these that are unresolved